### ISSUE TITLE
[BUG] HuggingFace Tool Connection Problem

### ISSUE THREAD

**Issue by MorphlingEd** [camel-ai/camel#268]
### Required prerequisites

- [X] I have read the documentation <https://camel-ai.github.io/camel/camel.html>.
- [X] I have searched the [Issue Tracker](https://github.com/camel-ai/camel/issues) and [Discussions](https://github.com/camel-ai/camel/discussions) that this hasn't already been reported. (+1 or comment there if it has.)
- [ ] Consider asking first in a [Discussion](https://github.com/camel-ai/camel/discussions/new).

### What version of camel are you using?

0.1.0

### System information

3.8.17 (default, Jul  5 2023, 21:04:15) [GCC 11.2.0] linux
0.1.0

### Problem description

The newly submitted commit at #245 indicates there occured problem in the tests of embodied agents and huggingface tools. The core error message is:
`urllib3.exceptions.NameResolutionError: <urllib3.connection.HTTPSConnection object at 0x7fade8303220>: Failed to resolve 'jwtq338l7l47q5zh.us-east-1.aws.endpoints.huggingface.cloud' ([Errno -2] Name or service not known)`
The 3 failed tests in `pytest_package_full_test` seem to have the similar problem. I guess this might be because HuggingFace changes their server settings. Hopefully this can be solved.

### Reproducible example code

pass

### Traceback

```pytb
urllib3.exceptions.MaxRetryError: HTTPSConnectionPool(host='jwtq338l7l47q5zh.us-east-1.aws.endpoints.huggingface.cloud', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7fade8303220>: Failed to resolve 'jwtq338l7l47q5zh.us-east-1.aws.endpoints.huggingface.cloud' ([Errno -2] Name or service not known)"))
```

**Comment by dandansamax** [comment]
Looks like the hugging face endpoint is down, which the hugging-face remote exection must invoke. Sadly, we can do nothing about that. I suggest we ignore this error until the hugging-face fix it.

**Comment by MorphlingEd** [comment]
Fixed by @dandansamax in #245
